bndtools.core/src/bndtools/launch/util/LaunchUtils.java

```diff
@@ -16,6 +22,16 @@
 
 public final class LaunchUtils {
 
+    private static final ILogger logger = Logger.getLogger(LaunchUtils.class);
+
+    private static final ServiceTracker<LaunchListener,LaunchListener> launchListeners;
+
+    static {
+        final BundleContext context = FrameworkUtil.getBundle(LaunchUtils.class).getBundleContext();
```

Comment:
Right now all callers to LaunchUtils are from other classes from the same bundle.  So I think we are ok?  However, we could add a ```if (!inited) then init()``` type of construct.  But would we need to protect that by a `synchronized` block?